ServiceNow Update to Yokohama work around (#31168)
hashicorp/vault: website/content/docs/platform/servicenow/configuration.mdx

1. Choose a type from the list.
1. Select "External credential store".
1. Provide a fully qualified collection name (FQCN):

>     - **Yokohama or newer**: You will need to enter in the credentials `com.snc.discovery.CredentialResolver`,
>         Under Credentials from the ServiceNow Webpage, you will be required to select your credentials and click 
>         the i for the mid server under “Credential Storage Vault” then click Open Record button to add the vault resolver FQCN

Reviewer: ```suggestion
    - **Yokohama or newer**: in the **Credentials** section of the ServiceNow
      webpage, select your credentials and click **ⓘ** for the "Credential
      Storage Vault" mid server. Click **Open Record** to add the vault resolver
      FQCN: `com.snc.discovery.CredentialResolver`
```
Style correction: write in active voice
